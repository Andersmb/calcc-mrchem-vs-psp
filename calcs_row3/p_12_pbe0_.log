Starting job 5688404 on c1-[24,36,38],c2-49,c3-18,c5-49 at Fri May 20 16:17:13 CEST 2022

Copying p_12_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/doublets
Copying p_12_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/doublets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688404      p_12_pbe0_        180                                             00:41:09      0:0 
5688404.bat+      batch         36        1   00:00:03          0   00:00:03   00:41:09      0:0 
5688404.ext+     extern        180        6   00:00:00          0   00:00:00   00:41:09      0:0 
5688404.0      mrchem.x        180       30   00:40:18         29   01:45:01   00:40:26      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688404                                                                          
5688404.bat+     19340K          0     19340K        0              0          0 
5688404.ext+          0          0          0        0              0          0 
5688404.0     12966468K         29 5142297395        0             11          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688404                                                                                               
5688404.bat+       19.44M               0         19.44M        0.07M                0          0.07M 
5688404.ext+        0.00M               0          0.00M            0                0              0 
5688404.0           6.77M               0          6.74M      125.86M                0         61.50M 

Job 5688404 completed at Fri May 20 16:58:23 CEST 2022
