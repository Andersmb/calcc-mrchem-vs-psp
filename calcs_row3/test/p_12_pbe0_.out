                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :        0.000000        0.000000        0.535625
    1       O            :        0.000000        2.563035       -0.518464
    2       O            :        0.000000       -2.563035       -0.518464
---------------------------------------------------------------------------
 Center of mass          :        0.000000       -0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              16
 Beta electrons          :                                              15
 Total electrons         :                                              31
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999911749e-01
    1    1    a          :                              9.999982196185e-01
    2    1    a          :                              9.999999474727e-01
    3    1    a          :                              1.000000001812e+00
    4    1    a          :                              9.999999997246e-01
    5    1    a          :                              9.999999991137e-01
    6    1    a          :                              9.999999999730e-01
    7    1    a          :                              9.999999646356e-01
    8    1    a          :                              1.000000056872e+00
    9    1    a          :                              9.999991864745e-01
   10    1    a          :                              1.000001184586e+00
   11    1    a          :                              9.999999694560e-01
   12    1    a          :                              1.000000116623e+00
   13    1    a          :                              1.000000286567e+00
   14    1    a          :                              9.999998289204e-01
   15    1    a          :                              9.999999675283e-01
   16    1    b          :                              9.999999885986e-01
   17    1    b          :                              9.999994092520e-01
   18    1    b          :                              1.000000697273e+00
   19    1    b          :                              1.000000001642e+00
   20    1    b          :                              1.000000000738e+00
   21    1    b          :                              9.999999991137e-01
   22    1    b          :                              9.999999999725e-01
   23    1    b          :                              9.999999646010e-01
   24    1    b          :                              1.000000047752e+00
   25    1    b          :                              9.999991863589e-01
   26    1    b          :                              1.000001183135e+00
   27    1    b          :                              9.999998593727e-01
   28    1    b          :                              1.000000116623e+00
   29    1    b          :                              1.000000324511e+00
   30    1    b          :                              9.999998289204e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                487.990428371063
 E-N energy              :            (au)              -1360.105688377411
 Coulomb energy          :            (au)                324.711056512577
 Exchange energy         :            (au)                 -9.826693499543
 X-C energy              :            (au)                -30.530390471056
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 99.086303686537
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -587.761287464370
 Nuclear energy          :            (au)                 99.086303686537
---------------------------------------------------------------------------
 Total energy            :            (au)             -4.886749837778e+02
                         :      (kcal/mol)             -3.066481795840e+05
                         :        (kJ/mol)             -1.283015984212e+06
                         :            (eV)             -1.329752314788e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -77.079698795548
    1    1    a          :            (au)                -18.823976128403
    2    1    a          :            (au)                -18.822945311319
    3    1    a          :            (au)                 -6.913912429151
    4    1    a          :            (au)                 -5.054237767235
    5    1    a          :            (au)                 -5.053913556608
    6    1    a          :            (au)                 -5.052761178990
    7    1    a          :            (au)                 -1.061303751447
    8    1    a          :            (au)                 -0.972339514151
    9    1    a          :            (au)                 -0.563541854380
   10    1    a          :            (au)                 -0.390266677983
   11    1    a          :            (au)                 -0.370634941159
   12    1    a          :            (au)                 -0.324592305811
   13    1    a          :            (au)                 -0.218628711359
   14    1    a          :            (au)                 -0.214979567060
   15    1    a          :            (au)                 -0.193592437422
   16    1    b          :            (au)                -77.076167561365
   17    1    b          :            (au)                -18.818135549129
   18    1    b          :            (au)                -18.817200591241
   19    1    b          :            (au)                 -6.908569601103
   20    1    b          :            (au)                 -5.050929615482
   21    1    b          :            (au)                 -5.049730273382
   22    1    b          :            (au)                 -5.045750519913
   23    1    b          :            (au)                 -1.038220725581
   24    1    b          :            (au)                 -0.952951292206
   25    1    b          :            (au)                 -0.432904501547
   26    1    b          :            (au)                 -0.379303036991
   27    1    b          :            (au)                 -0.346565812046
   28    1    b          :            (au)                 -0.294935480479
   29    1    b          :            (au)                 -0.208967323510
   30    1    b          :            (au)                 -0.206230053435
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -281.737886865435
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          5.567764e+00        -488.674983777833         -4.886750e+02
    1          4.542857e-01        -491.442014827083         -2.767031e+00
    2          2.021700e-01        -491.569768000932         -1.277532e-01
    3          1.054953e-01        -491.594860900873         -2.509290e-02
    4          5.317164e-02        -491.602900021615         -8.039121e-03
    5          5.914425e-02        -491.609698586950         -6.798565e-03
    6          1.701964e-02        -491.610470456336         -7.718694e-04
    7          1.621484e-02        -491.610970161793         -4.997055e-04
    8          7.194406e-03        -491.611042224394         -7.206260e-05
    9          5.707830e-03        -491.611079853065         -3.762867e-05
   10          3.090768e-03        -491.611093745467         -1.389240e-05
   11          3.514996e-03        -491.611102979888         -9.234421e-06
   12          1.948863e-03        -491.611106587058         -3.607170e-06
   13          1.614617e-03        -491.611108804621         -2.217564e-06
   14          1.349896e-03        -491.611110309572         -1.504951e-06
   15          7.179299e-04        -491.611110841738         -5.321658e-07
   16          5.057061e-04        -491.611111084770         -2.430326e-07
   17          3.291408e-04        -491.611111191628         -1.068572e-07
   18          2.711634e-04        -491.611111258964         -6.733649e-08
   19          1.727368e-04        -491.611111287920         -2.895558e-08
   20          1.157052e-04        -491.611111300453         -1.253306e-08
---------------------------------------------------------------------------
                      SCF converged in 20 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :        0.000000        0.000000        0.535625
    1       O            :        0.000000        2.563035       -0.518464
    2       O            :        0.000000       -2.563035       -0.518464
---------------------------------------------------------------------------
 Center of mass          :        0.000000       -0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                490.618301488302
 E-N energy              :            (au)              -1366.009135128937
 Coulomb energy          :            (au)                324.844919846183
 Exchange energy         :            (au)                 -9.766663603329
 X-C energy              :            (au)                -30.384837589210
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 99.086303686537
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -590.697414986990
 Nuclear energy          :            (au)                 99.086303686537
---------------------------------------------------------------------------
 Total energy            :            (au)             -4.916111113005e+02
                         :      (kcal/mol)             -3.084906274066e+05
                         :        (kJ/mol)             -1.290724785907e+06
                         :            (eV)             -1.337741924446e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -77.330403716300
    1    1    a          :            (au)                -19.266582748411
    2    1    a          :            (au)                -19.266573283978
    3    1    a          :            (au)                 -6.722520590671
    4    1    a          :            (au)                 -4.860585911841
    5    1    a          :            (au)                 -4.858957932135
    6    1    a          :            (au)                 -4.854208755353
    7    1    a          :            (au)                 -1.110451041836
    8    1    a          :            (au)                 -1.063770521402
    9    1    a          :            (au)                 -0.593270134839
   10    1    a          :            (au)                 -0.492508514033
   11    1    a          :            (au)                 -0.472339488202
   12    1    a          :            (au)                 -0.465835764187
   13    1    a          :            (au)                 -0.393878813072
   14    1    a          :            (au)                 -0.391997815989
   15    1    a          :            (au)                 -0.330474063354
   16    1    b          :            (au)                -77.327952511966
   17    1    b          :            (au)                -19.259300794899
   18    1    b          :            (au)                -19.259290681644
   19    1    b          :            (au)                 -6.718872132388
   20    1    b          :            (au)                 -4.857963521563
   21    1    b          :            (au)                 -4.852232429399
   22    1    b          :            (au)                 -4.852074886540
   23    1    b          :            (au)                 -1.090985160897
   24    1    b          :            (au)                 -1.042880449373
   25    1    b          :            (au)                 -0.560667445136
   26    1    b          :            (au)                 -0.457973767712
   27    1    b          :            (au)                 -0.457735323059
   28    1    b          :            (au)                 -0.450071990388
   29    1    b          :            (au)                 -0.390672994997
   30    1    b          :            (au)                 -0.376878923259
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -284.429912108825
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000000       -0.000030        0.826338
 Magnitude               :            (au)                        0.826338
                         :         (Debye)                        2.100341
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000       -0.261059
 Magnitude               :            (au)                        0.261059
                         :         (Debye)                        0.663545
---------------------------------------------------------------------------
 Total vector            :        0.000000       -0.000030        0.565279
 Magnitude               :            (au)                        0.565279
                         :         (Debye)                        1.436796
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 40m 19s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
